golang/src/com/google/idea/blaze/golang/resolve/BlazeGoImportResolver.java

```diff
@@ -215,7 +215,7 @@ public boolean isDirectory() {
     }
 
     @Override
-    public boolean processChildren(PsiElementProcessor<PsiFileSystemItem> psiElementProcessor) {
+    public boolean processChildren(PsiElementProcessor<? super PsiFileSystemItem> psiElementProcessor) {
```

Comment:
Can you try to build for `IJ-2020.2` after this change?
TBH I think this should be done in a compatible manner via sdk-compat probably (as I assume it will fail on 2020.2)

Reply:
Summarising an offline conversation with @mskonovalov:
It's not obvious to me why we need to do this the way upstream does to support older sdks. Maybe one day we might want to simultaneously support vX and vX-1 at the same time from the same branch.

@mskonovalov remembers @uri-canva wanting to do it this way last time, so there must be a good reason to do it! In any case, it's easy, so done! :)